flixel/math/FlxVector.hx

```diff
@@ -96,19 +96,18 @@ class FlxVector extends FlxPoint
 		return this;
 	}
 	
-	/**
-	 * Scale this vector.
-	 * 
-	 * @param	k - scale coefficient
-	 * @return	scaled vector
-	 */
-	public inline function scale(k:Float):FlxVector
-	{
-		x *= k;
-		y *= k;
-		return this;
-	}
-	
+    /**
+     * Scale this vector.
+     * 
+     * @param   k - scale coefficient
+     * @return  scaled vector
+     */
+    public inline override function scale(k:Float):FlxVector
```

Comment:
The problem is that `FlxPoint#scale()` is `inline` - you can't override a function that is inline, otherwise the compiles wouldn't be able to inline it anymore.

You definitely can't omit hthe override keyword.